backend/hub/models.py

```diff
@@ -498,10 +496,13 @@ def save(self, *args, **kwargs):
 
     def create_owner(self):
```

Comment:
Do we still need to automatically create an owner for an Organization?

Reply:
Leaving this as it is for now. I feel it's just a little bit risky.